### PR TITLE
Fix reading 4-bit data

### PR DESCRIPTION
**Pull request by jepler**
Related to https://github.com/adafruit/Adafruit_CircuitPython_ImageLoad/issues/47

An incorrect mask was being used when extracting 4-bit data from a file.

### CODE REVIEW

**Review by kmatch98 — APPROVED** [review]
Looks great.  You beat me to it!

File: shared-module/bitmaptools/__init__.c [hunk 1]
```diff
@@ -464,7 +464,7 @@ void common_hal_bitmaptools_readinto(displayio_bitmap_t *self, pyb_file_obj_t *f
                     int byte_offset = x / 2;
                     int bit_offset = 4 * (reverse_pixels_in_element ? (1 - x % 2) : x % 2);
 
-                    value = (rowdata8[byte_offset] >> bit_offset) & 7;
+                    value = (rowdata8[byte_offset] >> bit_offset) & 15;
```
Comment (deshipu) [inline shared-module/bitmaptools/__init__.c:467]: Would be more obvious if written as ``0x0f``.

**Review by FoamyGuy — APPROVED** [review]
I tested this successfully with a Feather RP2040 and the imageload simpletest script. Confirmed this does resolve the issue.

**Comment by jepler** [comment]
One action has been stuck in "upload artifacts" for 4 hours.  Gonna merge without green.

**Pull request merged**